README.rst

```diff
@@ -10,6 +10,16 @@ about writing one, please start reading at :pep:`1`. Note that the PEP Index
 (:pep:`0`) is automatically generated based on the metadata headers in other PEPs.
 
 
+Canonical links
+===============
+
+The canonical form of PEP links are zero-padded, such as
+https://peps.python.org/pep-0008/
+
+Shortcut redirects are also available.
+For example, https://peps.python.org/8 redirects to the canonical form.
```

Comment:
```suggestion
For example, ``https://peps.python.org/8`` redirects to the canonical link.
```

* Use a literal for something primarily intended as such
* Use more specific/direct verbiage